tst/CTA.Rules.Test/PortCoreTests.cs

```diff
@@ -163,7 +164,9 @@ public async Task TestWebApiWithReferencesUsingGenerator(string version)
                 }
             };
 
-            CodeAnalyzer analyzer = CodeAnalyzerFactory.GetAnalyzer(configuration, NullLogger.Instance);
+            //CodeAnalyzer analyzer = CodeAnalyzerFactory.GetAnalyzer(configuration, NullLogger.Instance);
```

Comment:
I looked at GetLanguageAnalyzerByProjectType and it's calling a specific factory for both VC and C# but those factories aren't really doing anything except newing up another object. Is there a plan to use those differently, or should we just try to rip out some of the factory stuff in general?

https://github.com/aws/codelyzer/blob/89b570063cb640d29002a050bc9ba6559d0951e8/src/Analysis/Codelyzer.Analysis/Analyzer/CSharpAnalyzerFactory.cs#L10

https://github.com/aws/codelyzer/blob/89b570063cb640d29002a050bc9ba6559d0951e8/src/Analysis/Codelyzer.Analysis/Analyzer/VBAnalyerFactory.cs#L10

Reply:
Sounds like there is already a plan for some refactoring.